src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,32 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck;
+        truck = new Truck();
 
+        Machine bulldozer = new Bulldozer();
+
+        Machine excavator = new Excavator();
+        //tablica z obiektami
```

Comment:
Please don't add redundant empty lines to your code. We don't need them after class declaration or method signature.